stable/instana-agent/templates/daemonset.yaml

```
@@ -0,0 +1,166 @@
+{{- if .Values.agent.key -}}
+{{- if .Values.zone.name -}}
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "instana-agent.baseName" . }}
+  labels:
+    {{- include "instana-agent.helmChartLabels" . | nindent 4 }}
+spec:
+  template:
+    metadata:
+      labels:
+        k8s.instana.io/name: {{ template "instana-agent.baseName" . }}
+{{- if .Values.agent.pod.annotations }}
+      annotations:
+{{ toYaml .Values.agent.pod.annotations | indent 8}}
```

This would be consistent with how you do it elsewhere:

```suggestion
        {{- toYaml .Values.agent.pod.annotations | nindent 8}}
```